aries-site/src/layouts/main/Layout.js

```diff
@@ -197,6 +203,9 @@ export const Layout = ({
                             <UpdateNotification name={title} />
                           )}
                           {children}
+                          {accessibilityInfo && (
```

Comment:
See comment left in anchor.mdx. Is there a way we can avoid adding this in the main layout? Including in the markdown file would make the content markup more readable and transparent. Implemented as is requires "insider knowledge" and is a bit opaque.